package.json

```diff
@@ -27,16 +27,16 @@
     "db:migrate-up-all": "npm run db:migrate",
     "db:migrate-down": "./src/server/database/migrate.sh down",
     "db:migrate-down-all": "./src/server/database/migrate.sh down-all",
-    "lint": "eslint 'src/**/*.js' 'tests/*.js' 'webpack/*.js'",
+    "lint": "eslint --fix 'src/**/*.js' 'tests/*.js' 'webpack/*.js'",
```

Comment:
Oooh! Danger zone. 🚨 

I'd rather this be invoked manually. I've seen this go **horribly** wrong before on other projects.